test/unit/attributes.js

```diff
@@ -1107,6 +1107,71 @@ QUnit.test( "val(select) after form.reset() (Bug #2551)", function( assert ) {
 	jQuery( "#kk" ).remove();
 } );
 
+QUnit.test( "select.val(space characters) (gh-2978)", function( assert ) {
```

Comment:
Well, there's the old way, the really old way, and the ancient way. I suspect you're talking about a newer way?